src/Platform/AcquiaCloudMultiSitePlatform.php

```diff
@@ -50,25 +50,37 @@ public function execute(Command $command, InputInterface $input, OutputInterface
     $sshUrl = $environment->sshUrl;
     [, $url] = explode('@', $sshUrl);
     [$application] = explode('.', $url);
+
     $sites = $this->getPlatformMultiSites($environment, $application, $output, $vendor_path[$env_id]);
     if (!$sites) {
       $output->writeln('<warning>No sites available. Exiting...</warning>');
       return 1;
     }
 
+    $group_name = $input->getOption('group');
     $input_uri = $input->getOption('uri');
+
+    if (!$input_uri && $group_name) {
+      $alias = $this->getAlias();
+      $platform_id = self::getPlatformId();
+      $sites = $this->filterSitesByGroup($group_name, $sites, $output, $alias, $platform_id);
+      if (empty($sites)) {
+        return 3;
+      }
+    }
+
     if ($input_uri) {
       if (in_array($input_uri, $sites, TRUE)) {
         $sites = [$input_uri];
       }
       else {
         $output->writeln(sprintf("Given Url does not belong to the sites within the platform. %s", $input_uri));
-        return 2;
+        return 4;
```

Comment:
Shouldn't this be 3 instead and the previous one 2? The current exit codes are: 1,3,4 why did we leave 2 out?